cinema/views.py

```diff
@@ -0,0 +1,42 @@
+from django.http import JsonResponse, HttpResponse
+from django.views.decorators.csrf import csrf_exempt
+from django.shortcuts import get_object_or_404
+from rest_framework.parsers import JSONParser
+from .models import Movie
+from .serializers import MovieSerializer
+
+@csrf_exempt
+def movie_list(request):
+    if request.method == 'GET':
+        movies = Movie.objects.all()
+        serializer = MovieSerializer(movies, many=True)
+        return JsonResponse(serializer.data, safe=False)
```

Comment:
Consider using Django REST Framework's `Response` class instead of `JsonResponse` for better integration with the framework's features and to handle content negotiation automatically.